Makefile

```diff
@@ -15,7 +15,7 @@
 SHELL = /bin/bash
 
 # MP CSI Driver version
-VERSION=1.13.0
+VERSION=0.6.0
```

Comment:
a downgrade? ?

Reply:
The version was for AWS CSI driver, we are still on 0.6